beacon-chain/rpc/prysm/v2/validator/sync_committee.go

```diff
@@ -187,13 +204,17 @@ func (vs *Server) GetSyncCommitteeContribution(
 			}
 		}
 	}
-	aggregatedSig := bls.AggregateSignatures(sigs)
+	aggregatedSig := make([]byte, 96)
+	aggregatedSig[0] = 0xC0
```

Comment:
We probably should define `0xC0` in params config

Reply:
I will add a follow up PR in `develop` to clean up the BLS stuff